watertap/unit_models/tests/test_osmotically_assisted_reverse_osmosis_0D.py

```diff
@@ -216,6 +217,24 @@ def test_option_pressure_change_calculated():
     assert isinstance(m.fs.unit.eq_area, Constraint)
 
 
+@pytest.mark.unit
+def test_option_friction_factor_spiral_wound():
+    m = ConcreteModel()
+    m.fs = FlowsheetBlock(dynamic=False)
+    m.fs.properties = props.NaClParameterBlock()
+    m.fs.unit = OsmoticallyAssistedReverseOsmosis0D(
+        property_package=m.fs.properties,
+        has_pressure_change=True,
+        concentration_polarization_type=ConcentrationPolarizationType.calculated,
+        mass_transfer_coefficient=MassTransferCoefficient.calculated,
+        pressure_change_type=PressureChangeType.calculated,
+        friction_factor=FrictionFactor.spiral_wound,
+    )
+
+    assert m.fs.unit.config.friction_factor == FrictionFactor.spiral_wound
+    assert isinstance(m.fs.unit.eq_friction_factor_spiral_wound, Constraint)
```

Comment:
My guess is that I just need to edit this line to `m.fs.???.eq_friction_factor_spiral_wound` since I don't think this constraint actually exists on the unit model (only in the membrane channel base), but I'm not sure what the "???" should be.